uiServices/src/testCaseServices_test.go

```diff
@@ -0,0 +1,51 @@
+package services
+
+import (
+	"github.com/go-chi/chi"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+
+func TestSuccessfulGetAllCases(t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	DirectoryPath := os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/"
+	request, err := http.NewRequest(http.MethodGet, "/test-cases", nil)
+
+	request.Header.Set("testCasePathDir", DirectoryPath)
+	request.Header.Get("testCasePathDir")
+
+	w := httptest.NewRecorder()
+	r.ServeHTTP(w, request)
+
+	if err != nil {
+		t.Error(err)
```

Comment:
you can replace this with 
```
if assert.NoError(err) { 
assert.Equal(t, http.StatusOK, w.Code, "Did not get all test cases")
}
```

Reply:
Should also have a tests checking the number of test cases returned and the response type e.g. can it be unmarshalled into []Case from the json